warp10/src/main/sh/package.sh

```diff
@@ -15,73 +15,104 @@
 #   limitations under the License.
 #
 
-#
-# Build the distribution .tgz for Warp 10
-#
+set -eu
+
+##
+## Build the Warp 10 distribution archive.
+## The archive generated is for installation and start up of a Warp 10 instance.
+##
+## This script includes the following steps:
+##  - Create the directory hierarchy
+##  - Copy startup scripts
+##  - Copy log4j, README, config, runner, bootstrap
+##  - Copy template configuration
+##  - Copy Warp 10 jar
+##  - Fix permissions
+##
 
 VERSION=$1
+if [ -z "${VERSION:+x}" ]; then
+  echo "Usage: $0 VERSION WARP_ROOT_PATH"
+  exit 1
+fi
+WARP10_HOME=warp10-${VERSION}
+
 # Warp 10 root project path (../warp10)
 WARP_ROOT_PATH=$2
-
-WARP10_HOME=warp10-${VERSION}
+if [ -z "${WARP_ROOT_PATH:+x}" ]; then
+  echo "Usage: $0 VERSION WARP_ROOT_PATH"
+  exit 1
+fi
+if [ ! -d "${WARP_ROOT_PATH}" ]; then
+  echo "${WARP_ROOT_PATH} folder does not exists."
+  exit 1
+fi
 
 ARCHIVE=${WARP_ROOT_PATH}/archive
 
 # Remove existing archive dir
-rm -rf ${ARCHIVE}
-
-# Create the directory hierarchy
-mkdir ${ARCHIVE}
-cd ${ARCHIVE}
-mkdir -p ${WARP10_HOME}/bin
-mkdir -p ${WARP10_HOME}/calls
-mkdir -p ${WARP10_HOME}/datalog
-mkdir -p ${WARP10_HOME}/datalog_done
-mkdir -p ${WARP10_HOME}/etc/bootstrap
-mkdir -p ${WARP10_HOME}/etc/conf.d
-mkdir -p ${WARP10_HOME}/etc/throttle
-mkdir -p ${WARP10_HOME}/etc/trl
-mkdir -p ${WARP10_HOME}/jars
-mkdir -p ${WARP10_HOME}/leveldb/snapshots
-mkdir -p ${WARP10_HOME}/lib
-mkdir -p ${WARP10_HOME}/logs
-mkdir -p ${WARP10_HOME}/macros
-mkdir -p ${WARP10_HOME}/templates
-mkdir -p ${WARP10_HOME}/runners/test/60000
+rm -rf "${ARCHIVE}"
 
+##
+## Create the directory hierarchy
+##
+mkdir "${ARCHIVE}"
+cd "${ARCHIVE}"
+mkdir -p "${WARP10_HOME}/bin"
+mkdir -p "${WARP10_HOME}/calls"
+mkdir -p "${WARP10_HOME}/datalog-ng"
+mkdir -p "${WARP10_HOME}/etc/bootstrap"
+mkdir -p "${WARP10_HOME}/etc/conf.d"
+mkdir -p "${WARP10_HOME}/etc/throttle"
+mkdir -p "${WARP10_HOME}/etc/trl"
+mkdir -p "${WARP10_HOME}/jars"
+mkdir -p "${WARP10_HOME}/lib"
+mkdir -p "${WARP10_HOME}/logs"
+mkdir -p "${WARP10_HOME}/macros"
+mkdir -p "${WARP10_HOME}/tokens"
+mkdir -p "${WARP10_HOME}/runners/test/60000"
```

Comment:
given there is a default runner with Sensision, do we retain the test one?

Reply:
Will be remove when sensision runner becomes available